### PR TITLE
added senior ui developer role

### PR DESCRIPTION
**Pull request by dkonieczek**

### CODE REVIEW

File: competencies/css-framework-bootstrap.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+## How do you prove it?
+
+You are able to integrate bootstrap into your application
+
+You are familiar with Bootstrap's grid system when using containers
+
+You are familiar with various Bootstrap components (ie. buttons, forms, modal)
+
+You are familiar with various Bootstrap utilities (ie. borders, colors, shadows)
+
```
Comment (codeallthethingz) [inline competencies/css-framework-bootstrap.md]: Build a simple form in Bootstrap

File: competencies/css-framework-tailwind.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+## How do you prove it?
+
+You are able to integrate Tailwind into your application
+
+Define the customization of your website in the Tailwind configuration file (ie. theme, breakpoints, colors, spacing)
+
+You are familiar with using Tailwind's layouts (ie. containers, display, float, position)
+
+You are familiar with display:flex and it's child properties as Tailwind is built with flexbox
+
```
Comment (codeallthethingz) [inline competencies/css-framework-tailwind.md]: Build a site and theme it with tailwindcss

File: roles/engineering-senior-ui-developer.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+Mentoring
+Meetings
+Customer Empathy
+Part of Successful Projects:2
+1 of IDE
+1 of Language
+2 of UI Framework
+1 of Design Tool
+1 of CSS Framework
+1 of CSS Preprocessor
```
Comment (codeallthethingz) [inline roles/engineering-senior-ui-developer.md]: I think sass or less won the war, and we should just say, you must have the winner.  Unless we're using them both.

**Comment by dkonieczek** [comment]
updated based on feedback: https://github.com/SearchSpring/competencies/pull/19/commits/fb64649c72e0b9296ba441f0cba908de68dce94f

**Pull request merged**